License2Deploy/rolling_deploy.py

```diff
@@ -292,6 +292,17 @@ def launch_new_instances(self, group_name): # pragma: no cover
       logging.error('Load balancer healthcheck has exceeded the timeout threshold. Rolling back.')
       self.revert_deployment()
 
+  def terminate_original_instances(self, group_name): #pragma: no cover
+    """ Will remove original instances in autoscale group """
+    logging.info("Removing old instances from autoscale group")
+    for instance_id in self.original_instance_ids:
+      try:
+        self.conn_auto.terminate_instance(instance_id, decrement_capacity=True)
+        logging.info("Removed {0} from autoscale group".format(instance_id))
+      except:
```

Comment:
It might be better to except Exception as e so e can get logged. 

Approving either way.